avalon/pipeline.py

```diff
@@ -720,6 +720,85 @@ def get_representation_context(representation):
     return context
 
 
+def update_current_task(task=None, asset=None, app=None):
+    """Update active Session to a new task work area.
+
+    This updates the live Session to a different `asset`, `task` or `app`.
+
+    Args:
+        task (str): The task to set.
+        asset (str): The asset to set.
+        app (str): The app to set.
+
+    Returns:
+        dict: The changed key, values in the current Session.
+
+    """
+
+    mapping = {
+        "AVALON_ASSET": asset,
+        "AVALON_TASK": task,
+        "AVALON_APP": app,
+    }
+    changed = {key: value for key, value in mapping.items() if value}
+    if not changed:
+        return
+
+    # Update silo when asset changed
+    if "AVALON_ASSET" in changed:
+        asset_document = io.find_one({"name": changed['AVALON_ASSET'],
+                                      "type": "asset"},
+                                     projection={"silo": True})
+        assert asset_document, "Asset must exist"
+        changed["AVALON_SILO"] = asset_document['silo']
+
+    # Compute work directory (with the temporary changed session so far)
+    project = io.find_one({"type": "project"},
+                          projection={"config.template.work": True})
+    template = project['config']['template']['work']
+    _session = Session.copy()
+    _session.update(changed)
+    changed['AVALON_WORKDIR'] = format_work_template(template, _session)
```

Comment:
Use double-ticks

Reply:
@mottosso across the file there are many lines that use single ticks - also on older lines. Do you want me to swap all?

Reply:
@mottosso are you able to take a look at this? :) I'd love to get this ready for merging.

Reply:
Just these for now is fine, but try and stay consistent.